scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ResumingClientHandshaker.java

```diff
@@ -159,6 +159,7 @@ protected void doProcessMessage(DTLSMessage message) throws HandshakeException,
 					// Server refuse to resume the session, go for a full handshake
 					fullHandshake  = true;
 					super.receivedServerHello(serverHello);
+					incrementNextReceiveSeq();
```

Comment:
Good catch !
Any reason to not just remove the `return;` ?

Reply:
With the return, it's clear the the control flow of the full handshake is used. 
Even if the current implementation would not differ.

Reply:
Ok but here we just want to increment and log we receive a message and this is was is done later :)